src/extension.ts

```diff
@@ -43,14 +43,15 @@ export function activate(context: vscode.ExtensionContext) {
       messages.push('Kindly provide a userToken')
     }
 
-    if (schemaURL && userName && userToken) {
+    if (jenkinsURL && userName && userToken) {
       // Fetches the JSON Schema via a REST API Call
       const auth = Buffer.from(`${userName}:${userToken}`).toString('base64')
       const options: RequestInit = {
         headers: {
           Authorization: `Basic ${auth}`,
         },
       }
+      const schemaURL = jenkinsURL + "configuration-as-code/schema"
```

Comment:
I think we should have an method that than adds the trailing slash 😓

Reply:
Sounds Cool to me :)